infra/gravity/node_commands.go

```diff
@@ -308,7 +349,16 @@ func (g *gravity) Remove(ctx context.Context, node string, graceful Graceful) er
 
 // Uninstall removes gravity installation. It requires Leave beforehand
 func (g *gravity) Uninstall(ctx context.Context) error {
-	cmd := fmt.Sprintf(`cd %s && sudo ./gravity system uninstall --confirm --system-log-file=./telekube-system.log`, g.installDir)
+	cmd := fmt.Sprintf(`cd %s && sudo ./gravity leave --force --confirm --system-log-file=./telekube-system.log`, g.installDir)
```

Comment:
For multi-node cluster this will launch shrink operation rather than just cleaning local data - is that intended?

Reply:
You're right, I'll revert.